### DISCUSSION TITLE
How to set a nullable field back to null?

### DISCUSSION THREAD

**Discussion by glenatron** [aarondl/sqlboiler#1394] [Q&A] [answered]
This might be somewhere in the documentation but I couldn't find it.

If I have a nullable field, say `null.String` and I have set it to a non-null value, how am I supposed to set the stored value back to `null` in the database?

**Comment by stephenafamo** [comment, marked as answer]
```go
s := null.StringFrom("hello") // non-null string
s.Valid = false // set to null
```
